terraform/trial_rg/services.tf

```diff
@@ -122,17 +128,32 @@ module "trial_app_service_init" {
     "WEBSITES_PORT"               = "8080" # The container is listening on 8080
   }
 
+
   depends_on = [
     azurerm_app_service_plan.apps_service_plan,
     module.trial_sc_config,
     module.trial_sc_discovery,
-    module.trial_app_service_site,
-    module.trial_app_service_practitioner,
-    module.trial_app_service_role,
     module.trial_sc_gateway,
   ]
 }
 
+data "azurerm_subscription" "current" {
+
+}
+
+data "azurerm_role_definition" "contributor" {
+  name = "Contributor"
+}
+resource "azurerm_role_assignment" "example" {
+  depends_on = [
+    module.trial_app_service_init
+  ]
+  skip_service_principal_aad_check = true
```

Comment:
what does this means?

Reply:
The skip_service..check is to ensure that teh AAD check doesnt fail when its a newly created principal (which it is, I think)..as per the docs..

(Optional) If the principal_id is a newly provisioned Service Principal set this value to true to skip the Azure Active Directory check which may fail due to replication lag.